js/app.js

```diff
@@ -0,0 +1,43 @@
+import { ToDoList  } from "./ToDoList.js";
+
+const taskInput = document.getElementById("task-input");
+const addbtn = document.querySelector(".input-text button");
+let selectedPrority =  document.getElementsByName("prority"); 
+
+
+
+
+
+  addbtn.addEventListener("click",()=>{
+    
+    const ToDolist = new ToDoList();  
+    let task = taskInput.value.trim(); 
+    let Prority ; 
+
+    for(let i =0;i<selectedPrority.length;i++){
+        if(selectedPrority[i].checked){
+           Prority =selectedPrority[i].value; 
+        }
+    }
+      
+
+    if (task == "") {
+      return;
```

Comment:
you can add an `alert("You should add a task!");` without return value as it's not needed